analysisCode/src/SmearedEvent.cpp

```diff
@@ -48,22 +48,43 @@ void SmearedEvent::setSmearedParticles()
       /// Skip the scattered electron, since it is special
       if(particle->GetE() == m_scatLepton->GetE())
 	continue;
-      if(m_verbosity > 0)
+      if(m_verbosity > 2)
 	{
-	  std::cout << "Truth  : "<<truthParticle->Id() 
+	  std::cout << "Truth (lab) : "<<truthParticle->Id() 
 		    << " " <<truthParticle->GetPx() << " " 
 		    << truthParticle->GetPy() << " " << truthParticle->GetPz()
 		    << " " << truthParticle->GetE() << std::endl;
 	  
-	  std::cout << "Smeared : " << particle->GetPx() << " " 
+	  std::cout << "Smeared (lab) : " << particle->GetPx() << " " 
 		    << particle->GetPy() << " " << particle->GetPz() << " " 
 		    << particle->GetE() << std::endl;
 	}
 
-      m_particles.push_back(fastjet::PseudoJet(particle->GetPx(),
-					       particle->GetPy(),
-					       particle->GetPz(),
-					       particle->GetE()));
+      // Transform Particle 4 Vectors to the Breit Frame 
+      TLorentzVector *truthPartFourVec = new TLorentzVector( truthParticle->PxPyPzE() );
+      labToBreit( truthPartFourVec, m_truthEvent);
+
+      TLorentzVector *partFourVec = new TLorentzVector( particle->PxPyPzE() );
+      labToBreit( partFourVec, m_truthEvent);
```

Comment:
This is going to boost the smeared particle to the breit frame based on the truth exchanged boson, correct? Should we boost by the reconstructed exchanged boson kinematics to be consistent?

Reply:
That is correct. This is a good question you bring up, my thought was that we should always be working in the "true" Breit frame but now I am not so sure. Do you know what would be more ideal in communicating with theorists? 

Implementing a function to boost by the smeared exchange boson kinematics would be a simple extension of this, I can just add another member function to the class (since I will make it into a class).

Reply:
I think we should have both as in actual measurements we won't know what the truth is, and part of the smearing process is to simulate what we will actually measure. So I think we should boost the truth particles by the truth exchanged boson, and boost smeared particles by the smeared exchanged boson kinematics (since that is in principle what we would measure).